static/app/components/demo/demoHeader.tsx

```diff
@@ -90,7 +90,7 @@ const Wrapper = styled('div')<{collapsed: boolean}>`
   text-transform: uppercase;
   align-items: center;
   white-space: nowrap;
-  gap: 3rem;
+  gap: ${space(4)};
```

Comment:
In the current scale, 3rem = 30px. This is also equal to `space(4)`, so we should use that instead.

<img width="948" alt="Screen Shot 2021-12-16 at 10 28 11 AM" src="https://user-images.githubusercontent.com/44172267/146444923-9915a09c-aa59-4a5e-884f-9a44fb942217.png">

Reply:
👍